File: cpp/c-plus-plus-find-factorial-class/example2.cpp
// https://codevscolor.com/c-plus-plus-find-factorial-class

#include <iostream>
using namespace std;

class Factorial
{
public:
    static unsigned long long calculateFactorial(unsigned long long num)
    {
        if (num == 0 || num == 1)
        {
            return 1;
        }
        return num * calculateFactorial(num - 1);
    }
};

int main()
{
    int num;

    cout << "Enter a number :" << endl;
    cin >> num;

    unsigned long long factorial = Factorial::calculateFactorial(num);
    cout << "Factorial: " << factorial << endl;
}

File: python/python-find-largest-divisor-number/example2.py
# https://codevscolor.com/python-find-largest-divisor-number
num = int(input("Enter a number: "))
largest_divisor = 0

for i in range(2, int(num/2+1)):
    if num % i == 0:
        largest_divisor = i

print("Largest divisor of {} is {}".format(num,largest_divisor))